tests/e2e/tutorials/tutorialBase.js

```diff
@@ -620,6 +623,53 @@ class TutorialBase {
     }
   }
 
+  async testS4LDipole(s4lNodeId) {
+    await this.waitFor(20000, 'Wait for the splash screen to disappear');
+
+    const s4lIframe = await this.getIframe(s4lNodeId);
+    await this.waitAndClick('mode-button-modeling', s4lIframe);
+    await this.waitAndClick('tree-model', s4lIframe);
+    await this.waitFor(2000, 'Model Mode clicked');
+    await this.takeScreenshot("Model");
+    const modelItems = await s4lIframe.$$('.MuiTreeItem-label');
+    console.log("N items in model tree:", modelItems.length/2); // there are 2 trees
+
+    await this.waitAndClick('mode-button-simulation', s4lIframe);
+    await this.waitFor(2000, 'Simulation Mode clicked');
+    await this.takeScreenshot("Simulation");
+
+    // click on simulation root element
+    const simulationsItems = await s4lIframe.$$('.MuiTreeItem-label');
```

Comment:
THOUGHT: It would be ideal to associate some sort of e2e scripts to a service upon publications (equivalent to the metadata, etc). This way we could have a generic way to run custom e2es on services that have iframes

Reply:
👍